front/src/app/navigation/navigationPage.ts

```diff
@@ -9,7 +9,6 @@ import {ActivatedRoute} from '@angular/router';
 import {SearchInputComponent} from './search-input/search-input.component';
 import {DrawerState} from '../../ionic-bottom-drawer/drawer-state';
 import {IonicBottomDrawerComponent} from '../../ionic-bottom-drawer/ionic-bottom-drawer.component';
-import { Storage } from '@ionic/storage';
```

Comment:
Hier ist beim merge irgendwas schief gegangen. Ich glaube auch, dass das ionic storage modul in den packages fehlt.